lib/puppet/provider/cs_location/crm.rb

```diff
@@ -18,7 +18,12 @@ def self.instances
     instances = []
 
     cmd = [ command(:crm), 'configure', 'show', 'xml' ]
-    raw, status = Puppet::Util::SUIDManager.run_and_capture(cmd)
+    if Puppet::PUPPETVERSION.to_f < 3.4
```

Comment:
In the scope of this PR, it's fine to lean towards consistency. But we need to make a follow-up effort to clean up the module.

There is probably not going to be a Puppet `3.10` but assuming there is, we'd absolutely need to fix this before that's released.